### PR TITLE
feat: add logging warning about bot secrets requirement for v8

### PR DESCRIPTION
**Pull request by klmcadams**
Add a logging warning for the actions that will require a bot-user and bot-email as inputs for v8. I'm not sure if this is the correct way to go about it, so please let me know if there are alternatives that make more sense. Thanks!

### CODE REVIEW

File: doc-changelog/action.yml [hunk 1]
```diff
@@ -121,6 +121,21 @@ runs:
...
+        level: "WARNING"
+        message: >
+          Ansys Actions v8 will require bot username and email inputs in the
+          ``doc-changelog``, ``doc-deploy-changelog``, ``doc-deploy-dev`` and
+          ``doc-deploy-stable`` actions. Please add the bot-user and bot-email inputs
+          to the respective workflows and see
+          https://actions.docs.ansys.com/version/dev/migrations/index.html#migration-guide
+          for more information.
+
+    # TODO: end
```
Comment (RobPasMue) [inline doc-changelog/action.yml:137]: Rather than a warning I would throw an error in case these values are not defined

Comment (RobPasMue) [reply doc-changelog/action.yml:137]: For example: https://github.com/ansys/actions/blob/3decf3fd44018dcaa9969b4d5145f7d5d55ed2a8/_doc-build-linux/action.yml#L273-L279

Comment (SMoraisAnsys) [reply doc-changelog/action.yml:137]: Agreeing with Roberto.

File: doc-changelog/action.yml [hunk 2]
```diff
@@ -121,6 +121,21 @@ runs:
       run: |
         python -m pip install --upgrade pip towncrier==${{ inputs.towncrier-version }} toml==${{ inputs.toml-version }}
 
+    # TODO: remove this deprecation in ansys/actions@v9
```
[the remark this answers is not in the document]
Comment (jorgepiloto) [reply doc-changelog/action.yml:124]: My fault, I miss-read and though we were in `v7.1`.

[the remark this answers is not in the document]
Comment (RobPasMue) [reply doc-changelog/action.yml:124]: Not yet

**Comment by klmcadams** [comment]
Tested the error logging:

[No secrets provided](https://github.com/ansys-internal/actions-sandbox/actions/runs/10925599174/job/30327489409?pr=92)
[Secrets provided](https://github.com/ansys-internal/actions-sandbox/actions/runs/10925632526/job/30327605227?pr=92)

**Review by RobPasMue — APPROVED** [review]

**Pull request merged**